### ISSUE TITLE
[Proofreading] Chapter 11: Morphic - Reproducibility of examples

### ISSUE THREAD

**Issue by phoeinx** [hpi-swa-lab/SqueakByExample-english#66]
I am proofreading Chapter 11: Morphic and am using the SWA20/21 image as my test image. (https://www.hpi.uni-potsdam.de/hirschfeld/artefacts/lecture-image/SWA2021.zip)

**Reproducibility of examples:**
- [ ] _R1. Page 222_ 
    - "Clicking on one of the yellow exclamation marks sends a message to the Morph which then responds appropriately." As can be seen in the picture below there are no yellow exclamation marks as far as I can see. And I think it would be helpful to give a starting explanation into how to switch categories to something like "fill & border" to have more fun attributes to play with.

  <img width="638" alt="image" src="https://user-images.githubusercontent.com/34538290/97631567-772ba200-1a31-11eb-9a41-768759627c64.png">

- [x]  _R2. Page 222_ 
    - "So, for example, Color tan asMorph returns a StringMorph labeled with the result of Color tan printString. Let’s change this so that we get a colored rectangle instead.
Open a browser on the Color class and add the following method to it:" 
        - In the current image the representation as a colored rectangle is already implemented in the standard Color class. 
  <img width="901" alt="image" src="https://user-images.githubusercontent.com/34538290/97631939-0d5fc800-1a32-11eb-85b3-dddba4a30808.png">

- [x] _R3. Page 224_
    - "From the world menu select “objects” and click on the button labeled “Graphics”. 
           - "objects" cannot be found on the first level anymore but under "old desktop menu..."
           - <img width="198" alt="image" src="https://user-images.githubusercontent.com/34538290/97691843-35394500-1a9f-11eb-913e-b0697669cd4c.png">

**Comment by codeZeilen** [comment]
Re `asMorph`: What about creating `Rectangle>>asMorph`? Not the most useful Morph but not too much in danger of being implemented in the image.

**Comment by codeZeilen** [comment]
> R1 (etoys examples): This was fixed via EToys-kfr.413 (Trunk). @codeZeilen Maybe should backport this to 5.3?

I could not find this commit @LinqLover. Is it -eem.413.mcz?

**Comment by LinqLover** [comment]
> > R1 (etoys examples): This was fixed via EToys-kfr.413 (Trunk). @codeZeilen Maybe should backport this to 5.3?
> 
> I could not find this commit @LinqLover. Is it -eem.413.mcz?

Sorry, I was wrong. The discussion thread is here: http://forum.world.st/The-Inbox-EToys-kfr-412-mcz-tp5123571p5124254.html ReleaseBuilder support does not exist as of today.

> Re `asMorph`: What about creating `Rectangle>>asMorph`? Not the most useful Morph but not too much in danger of being implemented in the image.

Hm ... sounds pretty useful, too, when you are debugging some Morphic code and would like to get an impression where some bounds would actually be :-) I think I'd like to propose this to the inbox ...

**Comment by codeZeilen** [comment]
I fixed the second issue.

The first issue is temporarily fixed by removing the passage from the text. However, I would prefer readding it later on.

The last one is the exact problem of mouse button mappings :) The world menu you opened is the wrong one...

**Comment by phoeinx** [comment]
well, ups. In my defense - the existence of two world menus is kinda confusing. :D

**Comment by LinqLover** [comment]
> well, ups. In my defense - the existence of two world menus is kinda confusing. :D

I think you were totally right. The other menu is labeled the "old world menu" but the book only mentioned a "world menu", which in consequence must be the "new world menu". 🤯 Maybe we should touch this in hpi-swa-lab/squeak#49 as well ...

**Comment by LinqLover** [comment]
I think we can mark issue 3 (wrong world menu) as done for now. In figure 1.3, the book explains correctly that the "normal" world menu is brought up via the red mouse button. IMHO this is not a problem with the book with the multiplicity of menus in Squeak. :-)

**Comment by LinqLover** [comment]
Ad R1 (viewer icons): This has been fixed in Squeak 6.0. However, I'm not sure whether we should introduce any Etoys concepts at all in the book, especially in this chapter. I'd opt for permanently removing this sentence (the current version of the book already refers to an explorer instead).

**Comment by LinqLover** [comment]
@codeZeilen Ad R1 (viewer icons): Currently, the example has been rewritten to describe an explorer instead of a viewer. Personally, I would vote for keeping it like this as I don't see the need to introduce Etoys concepts in the book. Also note that the original example works again in Squeak 6.0 and later.

tl;dr: Agree with closing this issue? :-)

**Comment by codeZeilen** [comment]
@LinqLover
